dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.internal.restfulWS-3.0.feature

```diff
@@ -15,7 +15,7 @@ Subsystem-Name: Internal Java RESTful Services 3.0
  com.ibm.websphere.appserver.org.eclipse.microprofile.config-1.4, \
  io.openliberty.jakarta.validation-3.0, \
  com.ibm.websphere.appserver.org.reactivestreams.reactive-streams-1.0, \
- com.ibm.websphere.appserver.mpConfig-1.4, \
+ com.ibm.websphere.appserver.org.eclipse.microprofile.config-1.4, \
```

Comment:
You already have that line

Reply:
Also, that feature depends on cdi-1.2 and tolerates 2.0 ... you're pulling in cdi-3.0